filament/backend/src/metal/MetalHandles.mm

```diff
@@ -143,6 +143,10 @@ static inline MTLTextureUsage getMetalTextureUsage(TextureUsage usage) {
     return (NSUInteger) layer.drawableSize.height;
 }
 
+#ifndef FILAMENT_LOCK_METAL_DRAWABLE_POOL
+#define FILAMENT_LOCK_METAL_DRAWABLE_POOL 1
+#endif
```

Comment:
do we need to make this optional?

Reply:
Was just following the same pattern as the release-on-main feature. It doesn't strictly need to be optional, I think. Will remove.